perun-db/postgres.sql

```diff
@@ -84,16 +84,16 @@ create table cabinet_publications (
 	id integer not null,
 	externalId integer not null, --identifier at externa pub. system
 	publicationSystemId integer not null, --identifier of external pub. system (cabinet_publication_systems.id) *)
-	title varchar(1024) not null,
+	title varchar not null,
 	year integer not null, --short title of publication
-	main varchar(4000), --full cite of publication
-	isbn varchar(32),
+	main varchar, --full cite of publication
+	isbn varchar,
 	categoryId integer not null, --identifier of category (cabinet_categories.id)
-	createdBy varchar(1300) default user not null,
+	createdBy varchar default user not null,
 	createdDate timestamp not null,
 	rank numeric (38,1) default 0 not null,
-	doi varchar(256),
-	locked varchar(1) default 0 not null,
+	doi varchar,
+	locked char(1) default 0 not null,
```

Comment:
Change ``varchar(1)`` to ``char(1)`` is missing in postgresql changelog. Also we should probably default to ``'0'`` (character not int) as other char columns do. It can be later updated to boolean, but I would like to have converts from char to boolean as a separate pull-request.

Reply:
I omitted it on purpose, I think that we do not need to migrate this column in the existing databases, it will be changed to boolean anyway.

Reply:
ok